content/developers/metrics/_index.md

```diff
@@ -120,6 +120,19 @@ The following units may be associated with metrics submitted to Datadog.
 | TEMPERATURE  | degree celsius / degree fahrenheit                                                                                                                                                                                                                 |
 | CPU          | nanocore / microcore / millicore / core / kilocore / megacore / gigacore / teracore / petacore / exacore                                                                                                                                           |
 
+Units are displayed automatically on timeseries graphs, query value widgets, and toplists, as shown in the screenshot of a Redis dashboard below:
+
+{{< img src="developers/metrics/redis_dash_metrics_units.png" alt="Redis dash metric units" responsive="true" style="width:70%;">}}
+
+On timeseries graphs, move your cursor over any graph to see the relevant units. The raw data is automatically converted to readable display units (fractions of a second to ms, millions of bytes per second to MiB/s, etc.):
+
+{{< img src="developers/metrics/postgres_commits.png" alt="postgres commits" responsive="true" style="width:70%;">}}
+
+Units are also displayed at the bottom of Timeboard graphs, and metric descriptions are available by selecting *Metrics Info* from the gear dropdown:
```

Comment:
Timeboard with a capital T - yes/no?

Reply:
We've been capitalizing Timeboard in the rest of the docs, so this seems okay. Whether we *should* be capitalizing Timeboard is a discussion for another day...